src/P3_terminal_velocity.jl

```diff
@@ -1,25 +1,21 @@
 """
-    ice_particle_terminal_velocity(p3, D, Chen2022, ρₐ, F_rim, th, use_aspect_ratio)
+    ice_particle_terminal_velocity(state, D, Chen2022, ρₐ, use_aspect_ratio)
```

Comment:
`state` might be confusing once we couple to the Atmos model, because there we use it to describe the whole model state (i.e. all prognostic variables). Maybe something like `p3_state`?